lib/decompress/huf_decompress.rs

```diff
@@ -623,15 +623,15 @@ unsafe extern "C" fn HUF_decompress4X1_usingDTable_internal_fast_c_loop(
     let mut ip = args.ip;
     let mut op = args.op;
 
-    assert!(cfg!(target_endian = "little"));
-    assert!(cfg!(target_pointer_width = "64"));
+    debug_assert!(cfg!(target_endian = "little"));
+    debug_assert!(cfg!(target_pointer_width = "64"));
```

Comment:
These can remain `assert!()`. They would get optimized away when on a platform where it doesn't trigger.